src/cript/nodes/primary_nodes/material.py

```diff
@@ -272,7 +272,7 @@ def parent_material(self) -> Optional["Material"]:
 
     @parent_material.setter
     @beartype
-    def parent_material(self, new_parent_material: "Material") -> None:
+    def parent_material(self, new_parent_material: Optional["Material"]) -> None:
```

Comment:
@InnocentBug would it be okay if this is allowed to be removed later in the script?

Reply:
From the scientific standpoint probably no.
Once a material is derived from another material, that won't change.

However, from a practical standpoint, it does make sense.
A parent material might have been added by mistake, or we copy this material (to make a new one) and that doesn't have parent.

So, I would leave it in.

Reply:
okie dokes, I agree with you, that is a very good point